### ISSUE TITLE
PPT must handle tracks starting at different times

### ISSUE THREAD

**Issue by IanMayo** [debrief/debrief#3149]
We have two tracks.  One starts later than the other.

When we replay the PPT, they both start at the same time, but one finishes earlier.

Clearly the start of the _shorter_ track should be delayed.  I'm _pretty sure_ we handled it in the python, but it certainly hasn't made it to the Java.

Steps to reproduce
* create modified version of `boat2.rep`.  Delete the first half of the points
* load `boat1.rep` and `boat2.rep` into a plot
* press `Record to PPT`, let it play forwards for a while.  The highlight on the `boat2.rep` track should not start moving until the relevant time
* finish exporting PPT, view in PowerPoint
* You'll see both tracks start at the same time.  The `boat2.rep` track start should be deferred.

**Comment by saulhidalgoaular** [comment]
Please, review the PR https://github.com/debrief/debrief/pull/3251
